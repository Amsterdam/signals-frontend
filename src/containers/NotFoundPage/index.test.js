/**
 * Testing the NotFoundPage
 */

import React from 'react';
import { shallow } from 'enzyme';
import { FormattedMessage } from 'react-intl';

import NotFound from './index';

describe('<NotFound />', () => {
  it('should render the Page Not Found text', () => {
    const wrapper = shallow(
      <NotFound />
    );
    expect(wrapper.contains(
      <h1>
        <FormattedMessage
          id="boilerplate.containers.NotFoundPage.header"
          defaultMessage={'Page not found.'}
        />
      </h1>)).toEqual(true);
  });
});
