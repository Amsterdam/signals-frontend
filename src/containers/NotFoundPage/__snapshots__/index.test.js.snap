// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render the Page Not Found text 1`] = `
<div
  className="container"
>
  <article>
    <h1>
      <FormattedMessage
        defaultMessage="Page not found."
        id="boilerplate.containers.NotFoundPage.header"
        values={Object {}}
      />
    </h1>
  </article>
</div>
`;
